### PR TITLE
Added seperator in view menu, added new help menu and moved version i…

### PR DESCRIPTION
**Pull request by Razzeee**
…nto it.

This is also nice to have in preparation of the auto update feature, as it needs a home and help is probably, where we would place it.

### CODE REVIEW

File: src/main/menus/app.js [hunk 1]
```diff
@@ -26,17 +26,6 @@ var createTemplate = function(mainWindow) {
   template.push({
     label: '&' + first_menu_name,
     submenu: [{
-      label: 'About ' + app_name,
```
Comment (yuya-oc) [inline src/main/menus/app.js]: Please keep the 'About Mattermost' item for OS X. #83

Comment (Razzeee) [inline src/main/menus/app.js]: Will do, just recently discovered that it's done that way on osx

**Comment by Razzeee** [comment]
Just updated the PR, as I have no mac, this is more or less untested and needs to be verified. Screenshot would also be nice.

**Comment by yuya-oc** [comment]
I have checked on Mac. Thanks!

**Pull request merged**